hazelcast/src/main/java/com/hazelcast/concurrent/lock/operations/LocalLockCleanupOperation.java

```diff
@@ -35,6 +35,14 @@
 
     private final String uuid;
 
+    /**
+     * This constructor should not be used to obtain an instance of this class; it exists to fulfill IdentifiedDataSerializable
+     * coding conventions.
+     */
+    public LocalLockCleanupOperation() {
+        uuid = "";
```

Comment:
Can't we use a pattern like
```
    public LocalLockCleanupOperation() {
        throw new UnsupportedOperationException("LocalLockCleanupOperation is local only.");
    }
```
for these classes? Your test could accept this as a valid implementation and there is no risk that the constructor is accidentally used (at least not in a test). The only issue could be that you won't be able to construct an instance to call `getId()`, right?